packages/node/example/bootstrappedApp/README.md

```diff
@@ -16,7 +16,7 @@ the build of the library. Then, run `npm ci` again, this time in
 ## Getting a token
 
 To get all the credentials needed to authenticate your script, you can `cd src/`,
-and then run `node bootstrap.js --oidcProvider <your identity provider>`. 
+and then run `node bootstrap.js --oidcIssuer <your identity provider>`. 
```

Comment:
As per comment on Engineering channel today (I know you're moving this repo, but I'm just noticing this here now):
```suggestion
and then run `node bootstrap.js --solidIdentityProvider <your Solid Identity Provider>`. 
```